.gitignore

```diff
@@ -14,3 +14,4 @@ packages
 .directory
 Thumbs.db
 *.userprefs
+*.ffs_db
```

Comment:
Could you add this in a separate commit (doesn't need to be a separate PR though) & say in the commit message what application generates it?